rwengine/src/audio/SoundManager.cpp

```diff
@@ -7,6 +7,45 @@
 #include <array>
 #include <iostream>
 
+SoundManager::SoundManager()
+{
+	initializeOpenAL();
+}
+
+SoundManager::~SoundManager()
+{
+	// De-initialize OpenAL
+	if(alContext) {
+		alcMakeContextCurrent(NULL);
+		alcDestroyContext(alContext);
+	}
+
+	if(alDevice)
+		alcCloseDevice(alDevice);
+}
+
+bool SoundManager::initializeOpenAL()
```

Comment:
@JayFoxRox I rearranged the constructor to be at the top of the file to be more consistent with the class member layout, then I added the destructor there.